src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -67,18 +82,22 @@ class Breadcrumb extends PureComponent {
     const homeUrl = createURL(null);
 
     return (
-      <div className={cssClasses.root}>
-        <a
-          className={homeClassNames.join(' ')}
-          href={homeUrl}
-          onClick={homeOnClickHandler}
-        >
-          <Template templateKey="home" {...this.props.templateProps} />
-        </a>
-        {breadcrumb}
+      <div className={rootClassnames}>
+        <ul className={cx(cssClasses.list)}>
+          <li className={homeClassnames}>
+            <a
```

Comment:
Template should be used instead of `a` so that we don't generate useless div or other container element.